naucse/models.py

```diff
@@ -599,7 +599,8 @@ class CourseLink(CourseMixin, Model):
     branch: str = DataProperty(link, default="master")
 
     info = ForkProperty(repo, branch, entry_point="naucse.utils.forks:course_info",
-                        args=lambda instance: [instance.slug])
+                        args=lambda instance: [instance.slug],
+                        timeout=1)
```

Comment:
OK, can we rather keep it default and override it in travis config if it takes to long?

Reply:
This argument is passed along to the Task instance, meaning it cannot be overridden by environment variables

Reply:
I see. Let's keep it this way then. However, why 1 sec? Can it fail on us on Travis if it is a bit slower sometime?

Reply:
The timeout is only applied to task execution itself, meaning excluding startup of the container, I find it hard to believe that once the container is started this particular function would fail to complete in a second - it only loads a file, deserializes it from YAML, takes a couple of values and returns them in a JSON.

That said I am not dead set on having this timeout, I'm fine with removing it and keeping the 5-sec default from Arca...

Reply:
I'm not dead set against having it 1 second for now. I'm just saying: Be prepared to make it less strict once it starts being problematic. I've seen Travis CI builds where it got stuck for 2 seconds without any obvious reason. Cloud is somebody else's computer, you never know what is running there  during our deployment.

Reply:
Ok, we'll see what happens once I release v0.2 of Arca and the build in this branch won't fail because it can't install Arca...

Reply:
Would you like to use a tarball requirement for now before you do a release?

Reply:
Timeout in tests :)

Reply:
Was just about to comment on it. I will revert it to the default...